host/sgx/sgxload.c

```diff
@@ -327,8 +327,11 @@ oe_result_t oe_sgx_create_enclave(
     uint64_t* enclave_addr)
 {
     oe_result_t result = OE_UNEXPECTED;
-    void* base = NULL;
+    void *base = NULL, *base_address = NULL;
```

Comment:
consider make base_address as uint64_t, which eliminates most of the type casting.

Reply:
When I tried changing type to uint64_t, it resulted in almost the same number of typecasting of base to (void *) since some of the functions expect it to be in uint64_t.